src/app/public/modules/search/search.component.scss

```diff
@@ -52,6 +52,14 @@
   }
 }
 
+// Search input (.sky-form-control) has z-index 2.
+// This ensures the button's focus border won't get clipped by input.
+.sky-search-input-container {
+  .sky-input-group-btn > button:focus {
```

Comment:
Anything wrong with applying the `z-index` with or without focus?

Reply:
We can get into tricky scenarios with dropdowns. For example, if a date-time picker is above another one, and you click to open the top dropdown, the bottom dropdown button will "break through" the dropdown and sit on top of it (because they both stem from the same z-index of 3). Because the dropdown is rendered after user interaction, the button takes precedence.

I chose to be even more specific with the `:focus` declaration to avoid nasty situations like this ☝️ .